content/en/dora_metrics/_index.md

```diff
@@ -222,12 +222,13 @@ Submit deployment events as described in [deployment frequency](#deployment-freq
 
 You are required to provide the following incident attributes:
 
-- `service`
+- `services` or `team`. At least one of them is required.
 - `started_at`
 
 You can optionally add the following incident attributes:
 
 - `finished_at` for *resolved incidents*. Required for [time to restore service](#time-to-restore-service).
+- `id` provide your own ID. Required to resolve a previously sent incident. When not provided the endpoint will return a random UUID that can be used in the next request.
```

Comment:
"`id` for identifying incidents when they are created and resolved. This attribute is user-generated; when not provided, the endpoint returns a Datadog-generated UUID."

re-worded a bit, is this accurate? I don't think we need to say "can be used in the next request"